packages/flutter/lib/src/rendering/object.dart

```diff
@@ -4,6 +4,7 @@
 
 /// @docImport 'dart:ui';
 ///
+/// @docImport 'package:flutter/rendering.dart';
```

Comment:
I am not too familiar with doc import style. should we import individual file instead given this file is in rendering library ?

Reply:
Ah, good catch. Yeah, just like regular imports, if it's from the same library we should import the file directly.